tests/test_rpc.py

```diff
@@ -6,17 +6,23 @@
 from xml.dom import minidom
 from pytest_rpc import ENV_VARS
 
+from dateutil import parser
```

Comment:
Ooo, that thing is nifty! However `parser` is really generic so you might want to import it with a more meaningful name:

```python
from dateutil import parser as date_parser
```